firestore.rules

```diff
@@ -0,0 +1,20 @@
+rules_version = '2';
+service cloud.firestore {
+  match /databases/{database}/documents {
+
+    // This rule allows anyone with your database reference to view, edit,
+    // and delete all data in your Firestore database. It is useful for getting
+    // started, but it is configured to expire after 30 days because it
+    // leaves your app open to attackers. At that time, all client
+    // requests to your Firestore database will be denied.
+    //
+    // Make sure to write security rules for your app before that time, or else
+    // all client requests to your Firestore database will be denied until you Update
+    // your rules
+    
+    match /{document=**} {
+      allow read, write: if
+          request.time < timestamp.date(2021, 6, 9);
```

Comment:
what should this rule be?  We probably want registered participants to be able to write.  So maybe we have one document set that's "participants" then another that is their responses and maybe only allow write to responses if the participant ID is in the participants db?  As an intermediate step, we can use `write: true` and `read: false`, but should definitely think about how to make that tighter (cc: @galenwinsor )

Reply:
I was able to write some firebase rules on the new branch.